src/selectors/entities/teams.js

```diff
@@ -42,6 +43,14 @@ export const getCurrentTeam = createSelector(
     }
 );
 
+export const getTeam = createSelector(
```

Comment:
This doesn't need to be a selector since it's just returning a part of the store

Reply:
yup, I'll remove the selector